Spine/Operation.swift

```diff
@@ -219,14 +219,17 @@ class SaveOperation: ConcurrentOperation {
 	/// Whether the resource is a new resource, or an existing resource.
 	private let isNewResource: Bool
 	
-	private let relationshipOperationQueue = NSOperationQueue()
+	private var relationshipOperationQueue: NSOperationQueue?
```

Comment:
I think it would be easier to just add the observer in the operation init method. The downside is that we create a queue even when it might not be necessary, but the big upside is that the variable becomes immutable which makes reasoning about the code a lot easier imho. Also, immutable variables reduces the chance to introduce bugs.